mygrad/math/arithmetic/ops.py

```diff
@@ -197,8 +197,9 @@ class MultiplySequence(BroadcastableOp):
     """ Performs f(a, b, ..., z) = a * b * ... * z"""
 
     def __call__(self, *input_vars):
-        assert len(input_vars) > 1, "`multiply_sequence` requires at least two operands"
         self.variables = input_vars
+        assert 2 <= len(self.variables)
```

Comment:
This should raise instead?

Reply:
It raises in the callable `multiply_sequence` function, as opposed to the operation.

A similar change should also be made to the `add_sequence` operation for consistency though, as
```python
assert len(input_vars) > 1, "`add_sequence` requires at least two operands"
```
is still run in `__call__` for the `AddSequence` class, despite the more descriptive `ValueError` now being raised in the `add_sequence` function.

Reply:
My concern is the `assert`s will be ignored in certain cases (e.g. when `__debug__` is `False`) and this won't do anything

Reply:
On second thought, I might like putting the input validation in the function and then using an assert to ensure consistency - this will allow for cleaner trace backs for users